Code/GraphMol/FMCS/Wrap/rdFMCS.cpp

```diff
@@ -59,6 +41,31 @@ MCSResult *FindMCSWrapper(python::object mols, bool maximizeBonds,
       p.BondTyper = MCSBondCompareOrderExact;
       break;
   }
+}
+
+MCSResult *FindMCSWrapper(python::object mols, bool maximizeBonds,
```

Comment:
Na, you can provide it with the parameter structure. I've decided that functions with 135 arguments are a bad idea. :-)